mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/InfrastructureTypeAnnotation.java

```diff
@@ -26,6 +26,6 @@ public String getName() {
 
   @Override
   public boolean isPublic() {
-    return false;
+    return true;
```

Comment:
I think that the super class/interface already defines a default implementation that returns true

Reply:
suggestion taken!